docs/mvvm/MigratingFromMvvmLight.md

```diff
@@ -0,0 +1,791 @@
+---
+title: Migrating from MvvmLight
+author: jamesmcroft
+description: This article describes how to migrate MvvmLight solutions to the Windows Community Toolkit MVVM framework.
+keywords: windows 10, uwp, windows community toolkit, uwp community toolkit, uwp toolkit, mvvm, mvvmlight, net core, net standard
+dev_langs:
+  - csharp
+---
+
+# Migrating from MvvmLight
+
+This article outlines some of the key differences between the [MvvmLight Toolkit](https://github.com/lbugnion/mvvmlight) and the MVVM Toolkit to ease your migration. 
+
+While this article specifically focuses on the migrations from MvvmLight to the MVVM Toolkit, note that there are additional improvements that have been made within the MVVM Toolkit so I highly recommend taking a look at the documentation for the APIs. 
+
+## Installing the WCT MVVM Toolkit
+
+To take advantage of the Windows Community Toolkit MVVM framework, you'll first need to install the latest NuGet package to your existing Windows application.
+
+### Install via .NET CLI
+
+```
+dotnet add package Microsoft.Toolkit.Mvvm --version x.x.x
+```
+
+### Install via PackageReference
+
+```
+<PackageReference Include="Microsoft.Toolkit.Mvvm" Version="x.x.x" />
+```
+
+## Migrating ObservableObject
+
+The following steps focus on migrating your existing components which take advantage of the `ObservableObject` of the MvvmLight Toolkit. 
+
+The Windows Community Toolkit MVVM framework provides an [`ObservableObject`](ObservableObject) type that is similar. 
+
+The first change here will be swapping using directives in your components.
+
+```csharp
+// MvvmLight
+using GalaSoft.MvvmLight;
+
+// Toolkit.Mvvm
+using Microsoft.Toolkit.Mvvm.ComponentModel;
+```
+
+Below are a list of migrations that will need to be performed if being used in your current solution.
+
+### ObservableObject Methods
+
+#### VerifyPropertyName ( string )
```

Comment:
Is there a specific reason why this ordering of APIs was chosen?
I was thinking we might want to put the other APIs first (eg. the `Set<T>(ref T, T, string)` method) instead of just starting off with a method we don't have and which completely lacks a replacement? 🤣

Reply:
Ah yeah sorry! I have no idea why I chose this ordering. In some cases, I was running through the API in MvvmLight and wrote them up in the order I came across them 🤣 

But yes, will re-order these to be more specific to the more important APIs. Alternatively, I could do them by their name order. Happy to take your steer on this, think that important APIs would be preferrable.

Reply:
Reshuffled these around now, prioritizing the SetProperty methods 😄